controller/user_controller.rb

```diff
@@ -25,6 +25,20 @@ def title
 			end
 		end
 
+		post '/update_hashtag' do
+			settings.log.info(params)
+			block_unsigned
+			return if params[:id].nil? || params[:value].nil? || !check_if_valide_hashtag?(params[:value])
+			id_hashtag = Hashtag.where(equality: {name: params[:value]})[0].id
```

Comment:
dangerous line because you use an index on the result of where without checking if there is anything in it

Reply:
PLUS it's a useless query because you already made a query on this Hashtag table in this controller